arch/arm/core/cortex_m/mpu/nxp_mpu.c

```diff
@@ -491,7 +494,7 @@ static int nxp_mpu_init(struct device *arg)
 	return 0;
 }
 
-#if defined(CONFIG_SYS_LOG)
+#if defined(CONFIG_LOG)
```

Comment:
I am not sure is this is still valid.
@nordic-krch: As far as I remember logger could accept messages very early, before kernel initialization. Is that still true?